mypy/nodes.py

```diff
@@ -481,9 +490,9 @@ def set_line(self, target: Union[Context, int], column: Optional[int] = None) ->
         self.variable.set_line(self.line, self.column)
 
 
-FUNCITEM_FLAGS = [
+FUNCITEM_FLAGS = FUNCBASE_FLAGS + [
```

Comment:
This change adds `is_property`, is this intentional?

Reply:
It is. I made this change partly on principle since `is_property` actually is a field of FuncBase -- it felt cleaner to just force all subclasses to preserve that field no matter what.

This change also doesn't actually change the serialized output in practice. FuncItem currently has only two subtypes: FuncDef and LambdaExpr. The former subclass previously explicitly set and serialized the `is_property` field so this change makes no difference there. The latter subclass never really uses `is_property` but also doesn't have any serialize/deserialize methods, which makes this change moot.